capstone-frontend/src/components/BookSearchTile.jsx

```diff
@@ -40,4 +30,31 @@ const BookSearchTile = (props) => {
   );
 }
 
+const BookListAddDropdown = (props) => {
+  if (props.userBookLists.length > 0) {
+    return (
+      <DropdownButton id="dropdown-list-add" className="dropdown-add" title="Add to List">
+        {
+          props.userBookLists.map(bookList =>
+            <Dropdown.Item key={bookList.id}
+              onSelect={() => addBookToBookList(props.book.id, bookList.id)}>
+              {bookList.name}
+            </Dropdown.Item>
+          )
+        }
+      </DropdownButton>
+    );
+  } else {
+    return (
+      <DropdownButton id="dropdown-list-add" className="dropdown-add" title="No Lists Found" variant="warning">
+        {
+          <Dropdown.Item href="/createlist" key="emptyBookList">
```

Comment:
I don't think you need `key` here since this isn't a list item.

Reply:
Fixed!